src/carrier/ela_carrier_impl.h

```diff
@@ -114,6 +115,7 @@ struct ElaCarrier {
     hashtable_t *friends;
 
     DStoreWrapper *dstorectx;
+    LMsgManager *lmsg_mgr;
```

Comment:
Why need "LMsgManager" specific structure to manage this. 
can be just like:
hashtable_t assmbly_bigmsg;
It should be similar to assembly invitation/response packet .
It sound to fit into current structure of this structure.